examples/platforms/posix/ble_nimble.c

```diff
@@ -0,0 +1,1111 @@
+/*
+ *
+ *    Copyright (c) 2018 Nest Labs, Inc.
```

Comment:
I am obviously fine with this license, though shouldn't we use OpenThread authors license?